railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Deprecate `bin/rake stats` in favor of `bin/rails stats`. #47713
```

Comment:
Thanks @JuanVqz, we usually don't link to the PR in the Changelog.
Sometimes we do link to an issue that is fixed, but that doesn't apply here.
```suggestion
*   Deprecate `bin/rake stats` in favor of `bin/rails stats`.
```